tox.ini

```
@@ -1,22 +1,22 @@
 [tox]
-envlist = py{27,34,35,py}-django{18,19,110},isort
+envlist = py{38, 39, 310, 311, py}-django{32,42},isort
```

we generally don't use spaces between the versions:

```suggestion
envlist = py{38,39,310,311,py}-django{32,42},isort
```

Also note that we tend to split this out for supported combinations *and* readability:

* py311-django32 is not supported, the newest Python version for Django 3.2 is Python 3.10

You can properly specify this as:

```ini
envlist = 
    py{38,39,310}-django{32,41,42}
    py311-django{41,42}
    isort
```

This gives you the following envs:

* py38-django32
* py38-django41
* py38-django42
* py39-django32
* py39-django41
* py39-django42
* py310-django32
* py310-django41
* py310-django42
* py311-django41
* py311-django42
* isort

Finally, I don't think we really need to care about PyPy, so we can drop the `py`